modules/cnv-confirming-policy-updates-on-nodes.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-configuring-node-network-policy
+
+[id="cnv-confirming-policy-updates-on-nodes_{context}"]
+= Confirming policy updates on nodes
+
+A `NodeNetworkConfigurationPolicy` manifest describes the desired network configuration on nodes. 
+When you apply a Policy manifest to the cluster, it creates a `NodeNetworkConfigurationPolicy` object for the whole cluster that represents the desired state and the actual status of the Policy on the cluster as a whole. 
```

Comment:
The `NodeNetworkConfigurationPolicy` is actually created directly by the user when they create the "Policy". The wording implies that it is created by someone else (at least to my non-native ears). The sentence is also little too long, for instance, I don't know to which part of it does "that represents the desired" belong.